cep-20.2.md

```diff
@@ -0,0 +1,711 @@
+  # host dependency
+  always_include_files: [path]
+
+  # wether to relocate binaries or not. If this is a list of paths, then
+  # only the listed paths are relocated
+  binary_relocation: bool (defaults to true) | [glob]
+
+  # force file to be detected as TEXT file (for prefix replacement)
+  has_prefix_files: [path]
+
+  # force file to be detected as BINARY file (for prefix replacement)
+  binary_has_prefix_files: [path]
+
+  # ignore all or specific files for prefix replacement
+  ignore_prefix_files: bool | [path] (defaults to false)
+
+  # wether to detect binary files with prefix or not
+  detect_binary_files_with_prefix: bool (defaults to true on Unix and (always) false on Windows)
+
+  # Wether to include the recipe or not in the final package
+  include_recipe: bool (defaults to true)
+
+  # environment variables to either pass through to the script environment or set
+  script_env: [env_vars]
+
+  # A run export adds a dependency to the run requirements of a package if listed in build or host dependencies
+  run_exports: [MatchSpec] OR {strong: [MatchSpec], weak: [MatchSpec], strong_constrains: [MatchSpec], weak_constrains: [MatchSpec], noarch: [MatchSpec]}
+
+  # Allow linking against libraries that are not in the run requirements
+  missing_dso_whitelist: [glob]
+
+  # Allow runpath / rpath to point to these locations outside of the environment
+  runpath_whitelist: [glob]
+
+  # This is only used in the pip feedstock.
+  disable_pip: bool (defaults to false)
+
+  # merge the build and host environments (used in many R packages on Windows)
+  merge_build_host: bool
+
+  # ignore run exports by name
+  ignore_run_exports: list
+
+  # ignore run exports coming from the specified packages
+  ignore_run_exports_from: list
+
+  # copies fn[.bat/.sh] to the appropriate location, adds `.bat` or `.sh` to the filename
+  pre-link: string
+  post-link: string
+  pre-unlink: string
+
+  # error out when overdepending
+  error_overdepending: bool (defaults to ?)
+  # error out when overlinking
+  error_overlinking: bool (defaults to ?)
+
+  # REMOVED:
+  # noarch_python: bool
+  # features: list
+  # msvc_compiler: str
+  # requires_features: dict
+  # provides_features: dict
+  # preferred_env: str
+  # preferred_env_executable_paths: list
+
+  # marked as "still experimental"
+  # pin_depends: Enum<"record" | "strict">
+  # overlinking_ignore_patterns: [glob]
+
+  # defaults to patchelf (only cudatoolkit is using `lief` for some reason)
+  # rpaths_patcher: None
+```
+
+## Source section
+
+```yaml
+source: [SourceElement]
+```
+
+where the different source elements are defined as follows.
+
+### URL source
+
+```yaml
+# url pointing to the source tar.gz|zip|tar.bz2|...
+url: url
+# destination folder in work directory
+folder: path
+# hash of the file
+sha256: hex string
+# absolute or relative path from recipe file
+patches: [path]
+# removed keys
+# fn: string
```

Comment:
We need to bring this one back as it can be useful for test files (the opencv-feedstock uses it to put a video in the right spot). We'll rename it to `file_name` though.